system/jlib/jtrace.cpp

```diff
@@ -1001,9 +1001,14 @@ class CInternalSpan : public CChildSpan
 class CClientSpan : public CChildSpan
 {
 public:
-    CClientSpan(const char * spanName, CSpan * parent)
+    CClientSpan(const char * spanName, CSpan * parent, const SpanTimeStamp * spanStartTimeStamp = nullptr)
     : CChildSpan(spanName, parent)
     {
+        if (spanStartTimeStamp && spanStartTimeStamp->isInitialized())
```

Comment:
if we add the starttime option to internal, it might make sense to move this to CChildSpan?